doc/src/stdlib/test.md

```diff
@@ -68,23 +68,25 @@ julia> @test foo(:cat) == 1
 Error During Test
   Test threw an exception of type MethodError
   Expression: foo(:cat) == 1
-  MethodError: `length` has no method matching length(::Symbol)
```

Comment:
No because the stacktrace is very weird from how Documenter evaluates things in its own module.